driver/GFDL/atmosphere.F90

```diff
@@ -582,8 +582,15 @@ subroutine atmosphere_dynamics ( Time, Surf_diff )
    if ( any((/ id_qdt_dyn, id_qldt_dyn, id_qidt_dyn, id_qadt_dyn /) > 0) .or. &
         query_cmip_diag_id(ID_tnhusa) ) qtend(:, :, :, 1:4) = Atm(mygrid)%q (isc:iec, jsc:jec, :, 1:4)
 
-   mw_air_store = Mw_air(sum(Atm(mygrid)%q(isc:iec,jsc:jec,:,1:3),dim=4)) !g/mol
+   mw_air_store = Mw_air(Atm(mygrid)%q (isc:iec, jsc:jec, :, sphum)   + &
```

Comment:
Note that this may need to be revised if another microphysics scheme with more microphysical species is used.